references/cli/adopt-templates/default.cue

```diff
@@ -126,7 +129,7 @@ import "list"
 		},
 		for r in resourceMap.workload + resourceMap.service {
 			type: "k8s-objects"
-			name: "\(r.kind).\(r.metadata.name)"
+			name: strings.ToLower("\(r.kind).\(r.metadata.name)")
```

Comment:
The dot should not be there as well.

Reply:
I replaced `.` with `-` in my new commit.